x/ratelimit/keeper/rate_limit.go

```diff
@@ -1,13 +1,64 @@
 package keeper
 
 import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/ibc-go/v3/modules/core/exported"
 
 	"github.com/Stride-Labs/stride/v4/x/ratelimit/types"
 )
 
-func CheckRateLimit(direction types.PacketDirection, packet exported.PacketI,
-) error {
+func CheckRateLimit(direction types.PacketDirection, packet exported.PacketI) error {
 	// TODO
 	return nil
 }
+
+// Stores/Updates a rate limit object in the store
+func (k Keeper) SetRateLimit(ctx sdk.Context, rateLimit types.RateLimit) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.RateLimitKey))
+
+	rateLimitKey := types.KeyPrefix(rateLimit.Path.Id)
+	rateLimitValue := k.cdc.MustMarshal(&rateLimit)
+
+	store.Set(rateLimitKey, rateLimitValue)
+}
+
+// Removes a rate limit object from the store using the PathId
+func (k Keeper) RemoveRateLimit(ctx sdk.Context, pathId string) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.RateLimitKey))
+	rateLimitKey := types.KeyPrefix(pathId)
+	store.Delete(rateLimitKey)
+}
+
+// Grabs and returns a rate limit object from the store using the PathId
+func (k Keeper) GetRateLimit(ctx sdk.Context, pathId string) (rateLimit types.RateLimit, found bool) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.RateLimitKey))
+
+	rateLimitKey := types.KeyPrefix(pathId)
+	rateLimitValue := store.Get(rateLimitKey)
+
+	if rateLimitValue == nil || len(rateLimitValue) == 0 {
```

Comment:
This checking causes a golang lint error.
`rateLimitValue == nil` is same as `len(rateLimitValue) == 0` so its ok to check only nil value.